Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 1117, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/util.py", line 78, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/util.py", line 57, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 559, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 862, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 765, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
te1_img = "../data/sub-04570/func/sub-04570_task-rest_echo-1_space-scanner_desc-partialPreproc_bold.nii.gz"
te2_img = "../data/sub-04570/func/sub-04570_task-rest_echo-2_space-scanner_desc-partialPreproc_bold.nii.gz"
mask_img = "../data/sub-04570/func/sub-04570_task-rest_space-scanner_desc-brain_mask.nii.gz"
denoised_img = regress_one_image_out_of_another(te2_img, te1_img, mask_img)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
[0;32m/tmp/ipykernel_1791/2862625900.py[0m in [0;36m<module>[0;34m[0m
[1;32m      2[0m [0mte2_img[0m [0;34m=[0m [0;34m"../data/sub-04570/func/sub-04570_task-rest_echo-2_space-scanner_desc-partialPreproc_bold.nii.gz"[0m[0;34m[0m[0;34m[0m[0m
[1;32m      3[0m [0mmask_img[0m [0;34m=[0m [0;34m"../data/sub-04570/func/sub-04570_task-rest_space-scanner_desc-brain_mask.nii.gz"[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 4[0;31m [0mdenoised_img[0m [0;34m=[0m [0mregress_one_image_out_of_another[0m[0;34m([0m[0mte2_img[0m[0;34m,[0m [0mte1_img[0m[0;34m,[0m [0mmask_img[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m/tmp/ipykernel_1791/2172515877.py[0m in [0;36mregress_one_image_out_of_another[0;34m(data_img, nuis_img, mask_img)[0m
[1;32m      6[0m [0;32mdef[0m [0mregress_one_image_out_of_another[0m[0;34m([0m[0mdata_img[0m[0;34m,[0m [0mnuis_img[0m[0;34m,[0m [0mmask_img[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m      7[0m     [0;31m# First, mean-center each image over time[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 8[0;31m     [0mmean_data_img[0m [0;34m=[0m [0mimage[0m[0;34m.[0m[0mmean_img[0m[0;34m([0m[0mdata_img[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      9[0m     [0mmean_nuis_img[0m [0;34m=[0m [0mimage[0m[0;34m.[0m[0mmean_img[0m[0;34m([0m[0mnuis_img[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m     10[0m [0;34m[0m[0m

[0;32m/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nilearn/image/image.py[0m in [0;36mmean_img[0;34m(imgs, target_affine, target_shape, verbose, n_jobs)[0m
[1;32m    549[0m [0;34m[0m[0m
[1;32m    550[0m     [0mimgs_iter[0m [0;34m=[0m [0miter[0m[0;34m([0m[0mimgs[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 551[0;31m     [0mfirst_img[0m [0;34m=[0m [0mcheck_niimg[0m[0;34m([0m[0mnext[0m[0;34m([0m[0mimgs_iter[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    552[0m [0;34m[0m[0m
[1;32m    553[0m     [0;31m# Compute the first mean to retrieve the reference[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m

[0;32m/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nilearn/_utils/niimg_conversions.py[0m in [0;36mcheck_niimg[0;34m(niimg, ensure_ndim, atleast_4d, dtype, return_iterator, wildcards)[0m
[1;32m    269[0m                 [0;32mraise[0m [0mValueError[0m[0;34m([0m[0mmessage[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    270[0m             [0;32melse[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 271[0;31m                 [0;32mraise[0m [0mValueError[0m[0;34m([0m[0;34m"File not found: '%s'"[0m [0;34m%[0m [0mniimg[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    272[0m         [0;32melif[0m [0;32mnot[0m [0mos[0m[0;34m.[0m[0mpath[0m[0;34m.[0m[0mexists[0m[0;34m([0m[0mniimg[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    273[0m             [0;32mraise[0m [0mValueError[0m[0;34m([0m[0;34m"File not found: '%s'"[0m [0;34m%[0m [0mniimg[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;31mValueError[0m: File not found: '../data/sub-04570/func/sub-04570_task-rest_echo-2_space-scanner_desc-partialPreproc_bold.nii.gz'
ValueError: File not found: '../data/sub-04570/func/sub-04570_task-rest_echo-2_space-scanner_desc-partialPreproc_bold.nii.gz'

